cryptofeed/backends/backend.py

```diff
@@ -12,49 +12,48 @@
 
 
 class BackendBookCallback:
```

Comment:
It looks like data always has `timestamp` and `receipt_timestamp`. I don't think your `write` functions need it explicitly passed since if it needs it, it's always accessible in the data

Reply:
they all take the same args as the user defined callbacks, so it has to be in the args. Its a lot easier to just dump it in the data than to re-process the data with that info (in my opinion). some of them do just use it from the write as opposed to the data.